ansible/roles/providers/tasks/aws.yml

```diff
@@ -19,11 +19,27 @@
   when:
     - ansible_os_family == "Debian"
 
+- name: install cython for flatcar
+  pip:
+    name: "cython<3.0.0"
+    executable: pip3
+    extra_args: "wheel"
+  when:
+    - ansible_distribution == "Flatcar"
+
+- name: install pyyaml
+  pip:
+    name: "pyyaml==5.4.1"
+    executable: pip3
+    extra_args: "--no-build-isolation"
+  when:
+    - ansible_distribution == "Flatcar"
+
 - name: install aws clients
   pip:
     name: "awscli"
```

Comment:
originally i tried pinning the version to 1.29.9 like we have in requirements-devkit.txt but this didn't work.

i then tried to see if i could use a newer pypy version https://www.pypy.org/download.html but this didn't work either.